common/lib/xmodule/xmodule/tests/test_editing_module.py

```diff
@@ -22,8 +22,7 @@ class TabsEditingDescriptorTestCase(unittest.TestCase):
 
     def setUp(self):
         super().setUp()
-        system = get_test_descriptor_system()
-        system.render_template = Mock(return_value="<div>Test Template HTML</div>")
+        system = get_test_descriptor_system(render_template=Mock(return_value="<div>Test Template HTML</div>"))
```

Comment:
Is a custom mock needed here or will a default be fine?

Reply:
Ah yep! simple Mock() works fine here.